public/app/core/angular_wrappers.ts

```diff
@@ -36,12 +37,12 @@ const { SecretFormField } = LegacyForms;
 export function registerAngularDirectives() {
   react2AngularDirective('footer', Footer, []);
   react2AngularDirective('icon', Icon, [
-    'color',
```

Comment:
We don't have color prop. I have probably forgotten to remove this when implementing Icon component.